src/smdba/postgresqlgate.py

```diff
@@ -605,6 +605,11 @@ def _rst_replace_new_backup(self, backup_dst: str) -> None:
         """
         Replace new backup.
         """
+        destination_tar = backup_dst + "/base.tar.gz"
```

Comment:
I see this is just a refactor, but find this naming really confusing.

We are trying to restore a backup from a file, but we use a variable called `destination_tar` :-?

Reply:
not nice, but tbh I don't want to include a name change refactor at this point. I just move this because I needed to add an "if" that needs this variable, nothing else.